doc/tools/make_rst.py

```diff
@@ -992,9 +1068,26 @@ def make_rst_class(class_def: ClassDef, state: State, dry_run: bool, output_dir:
 
             if theme_item_def.text is not None and theme_item_def.text.strip() != "":
                 f.write(f"{format_text_block(theme_item_def.text.strip(), theme_item_def, state)}\n\n")
+            else:
+                f.write(".. container:: contribute\n\n\t")
+                f.write(
+                    translate(
+                        "There is currently no description for this theme property. Please help us by :ref:`contributing one <doc_updating_the_class_reference>`!"
+                    )
+                    + "\n\n"
+                )
 
             index += 1
 
+    if not has_any_content:
```

Comment:
I wonder if instead of all these `has_any_content` assignments, it wouldn't be simpler to just treat the description like we do with properties/methods and always show this call to action when empty, even if some other parts of the API are documented?

Reply:
Yeah, I could reduce this to displaying the note if a class has neither full nor brief description. But if either one of these exists, I think the note should be hidden, as I don't think we have a rule requiring either one of them specifically.

Reply:
Done that.